testrunner/FFOSBR.js

```diff
@@ -12,67 +12,78 @@
 
 },{"./settings":2}],2:[function(require,module,exports){
 function Settings() {
-  var timeInMilliSec = 0;
 
   var options = {
-    photos: false,
-    videos: false,
-    contacts: false,
-    text: false,
-    intervalTime: 0, // pass in value in hours
+    photos: true,
+    videos: true,
+    contacts: true,
+    text: true,
+    intervalTime: 24, // pass in value in hours
     id: 0,
     registeredTimer: false,
-    repeat: false
+    repeat: true
   };
 
+  // Load options if present
```

Comment:
Ok. I think I see the issue here. Just so you, none of the options are being saved to the Settings object. Meaning, "Settings.options" is undefined. I think that's also why there's confusion about needing the ASCF. Anything declared as "var foo" in a constructor only exists in the constructor. Anything declared as "this.foo" is attached to the object. This would explain why the prototypes had to be defined within the constructor, which are typically defined outside the constructor, otherwise they are redefined every time a new instance of the object is made. I'm going to do a quick refactor if that's ok with you. The changes are minor.